pypi/build.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/sh
+
+mkdir -p /build
+cd /build
+
+echo "Hello from PyPi build.sh"
+
+BUILD_DIR=`pwd`
+if test -f /proc/cpuinfo; then
+  echo "cpuinfo exists"
+  N_CORES=`cat /proc/cpuinfo | grep processor | wc -l`
+else
+  echo "cpuinfo DOES NOT exist"
+  N_CORES=1
+fi
+
+CMAKE_DIR=cmake-3.5.0-Linux-x86_64
+
+if test ! -f ${CMAKE_DIR}.tar.gz; then
+  curl -L -O https://cmake.org/files/v3.5/${CMAKE_DIR}.tar.gz
+  if test $? -ne 0; then exit 1; fi
+fi      
+
+cp -r /boolector .
+
+if test ! -f lingeling-master.zip; then
+  curl -L -o lingeling-master.zip https://github.com/arminbiere/lingeling/archive/master.zip
+  if test $? -ne 0; then exit 1; fi
+fi
```

Comment:
Was there a specific reason why you didn't use the scripts in `contrib/` for setting up the dependencies? The issue with this is that you are always downloading master, but the `contrib` scripts define specific solver versions in order to not break the builds in case something changes upstream in the dependencies.

Reply:
Yes. The commit of lingeling that Boolector uses causes a compile error with the GCC used for Python. This has been fixed on 'master'. Perhaps we could update to using a newer commit?

Reply:
BTW, I've updated the lingeling commit in the PR